src/main/java/io/sealights/plugins/sealightsjenkins/buildsteps/cli/SealightsCLIBuildStep.java

```diff
@@ -33,6 +42,66 @@ public SealightsCLIBuildStep(boolean enabled, boolean failBuildIfStepFail,
         this.cliRunner = cliRunner;
     }
 
+    /* * The goal of this method is to support migration of data between versions
+    * of this plugin.
+    */
+    private Object readResolve() {
+        SealightsCLIBuildStep _this = this;
+        if (cliRunner != null) {
+            _this = resolveFromCLiRunner();
+        }
+        if(commandMode instanceof CommandMode.ConfigView){
+            _this =resolveTechOption();
+        }
+        return _this;
+    }
+
+    private SealightsCLIBuildStep resolveTechOption(){
+        String packagesIncluded = ((CommandMode.ConfigView) commandMode).getPackagesIncluded();
+        String packagesExcluded = ((CommandMode.ConfigView) commandMode).getPackagesExcluded();
+        DescribableList<TechnologyOptions, TechnologyOptionsDescriptor> technologyOptions = new DescribableList<>(JavaOptions.DescriptorImpl.NOOP);
+        technologyOptions.add(new JavaOptions(packagesIncluded,packagesExcluded));
+        ((CommandMode.ConfigView) commandMode).setTechOptions(technologyOptions);
+        return this;
+    }
+
+    private SealightsCLIBuildStep resolveFromCLiRunner() {
+        StringBuilder  additionalArgs = new StringBuilder();
+        if (this.commandMode instanceof CommandMode.ConfigView) {
+               commandMode =setConfigparams();
```

Comment:
A method which is setting something, shouldn't return a value. So, I assume that 'setConfigparams' should be called 'getConfigParams'.